bindings/rascal/models/kernels.py

```diff
@@ -107,9 +107,21 @@ def _get_init_params(self):
 
     def _set_data(self, data):
         super()._set_data(data)
+        # allows to load deprecated models
+        if "cpp_kernel" in data.keys():
+            self._kernel = self._kernel.from_dict(data["cpp_kernel"])
+        else:
+            print(
```

Comment:
Consider replacing with `LOGGER.warn()`  -- NB needs the following near the imports:
```
import logging
LOGGER = logging.getLogger(__name__)
```
(see e.g. `bindings/rascal/utils/filter.py` for an example of how this can be done)

Reply:
added LOGGER